package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "mocha --require ts-node/register --extension ts ./src --recursive",
     "test:watch": "mocha --require ts-node/register --extension ts ./src --recursive --watch",
-    "lint": "eslint ./src/**/*.ts",
-    "lint:fix": "eslint ./src/**/*.ts --fix",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
```

Comment:
I don't know why, but `./src/**/*.ts` only found the files in subfolders of `src`, but not in `src` directly, like `src\index.ts`.
To fix this, I included the whole repo via `.`.

Reply:
![image](https://github.com/user-attachments/assets/11a655c0-4f6e-449d-9c8d-f6fa6a79130a)